lib/gui/pages/main/controllers/main.js

```diff
@@ -65,4 +66,22 @@ module.exports = function(
     return this.shouldDriveStepBeDisabled() || !SelectionStateModel.hasDrive();
   };
 
+  /**
+   * @summary Open a drive info modal
+   * @function
+   * @public
+   *
+   * @returns {Boolean} whether the flash step should be disabled
+   *
+   * @example
+   * if (MainController.shouldFlashStepBeDisabled()) {
+   *   console.log('The flash step should be disabled');
+   * }
+   */
+  this.openDriveInfo = () => {
+    return DriveInfoModalService.display({
```

Comment:
Oh, I see it reads from `SelectionStateModel` internally. A better design would be to decouple `DriveInfoModal` from `SelectionStateModel` and pass the drive to it from here.